maestro-test/src/main/kotlin/maestro/test/drivers/FakeDriver.kt

```diff
@@ -358,6 +357,12 @@ class FakeDriver : Driver {
         )
     }
 
+    override fun setPermissions(appId: String, permissions: Map<String, String>) {
+        ensureOpen()
+
+        events.add(Event.SetPermissions(appId, permissions))
```

Comment:
Can we add IntegrationTest for this?

Reply:
Sure, but lets first make sure that the direction is approved and the code works on both platforms.

Reply:
I'll add the integration test now.

Reply:
Integration tests don't cover the code with logic in this PR. I've added one, but its not really testing anything.